recipes/onetbb/all/test_package/conanfile.py

```diff
@@ -7,10 +7,15 @@
 class TestPackageConan(ConanFile):
     settings = "os", "arch", "compiler", "build_type"
     generators = "CMakeToolchain", "CMakeDeps", "VirtualRunEnv"
+    should_skip_executing = False
```

Comment:
this would be a first in CCI so I am not sure this is the best way since consumer would have the same problem... 

Why publish a package that we know does not work?

Reply:
Yeah, I think it's important this be tested as part of the test package. TBB underpins STL Parallel Algorithms in GCC, unless that's changed since I used it. I think it's critical that we verify that the runtime is integrated properly.